integration_tests/test_gated.py

```diff
@@ -0,0 +1,44 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2016 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+
+import integration_tests
+
+
+class GatedTestCase(integration_tests.StoreTestCase):
+
+    def setUp(self):
+        if os.getenv('TEST_STORE') != 'fake':
+            self.skipTest('Right combination of snaps and IDs is not '
+                          'available in real stores.')
```

Comment:
So, we will never be able to test this against a real system? There is no way to seed some valid values there?